pkg/transform/report_output.go

```diff
@@ -69,6 +72,13 @@ type RouteReport struct {
 	WildcardPolicy    O7tapiroute.WildcardPolicyType     `json:"wildcardPolicy"`
 }
 
+// ContainerResourcesReport represents json report for aggregated container resources
+type ContainerResourcesReport struct {
+	ContainerCount int64              `json:"containerCount"`
+	CPUsummary     *resource.Quantity `json:"cpuSummary"`
```

Comment:
What is the need of having `CPUsummary ` as resource.Quantity in final report? Why not store it string?